AEPLifecycle/Sources/LifecycleV2/LifecycleV2.swift

```diff
@@ -28,10 +30,12 @@ class LifecycleV2 {
     /// Creates a new `LifecycleV2` with the given `NamedCollectionDataStore`
     ///
     /// - Parameter dataStore: The `NamedCollectionDataStore` used for reading and writing data to persistence
-    init(dataStore: NamedCollectionDataStore) {
+    init(dataStore: NamedCollectionDataStore, metricsBuilder: LifecycleV2MetricsBuilder? = nil, dispatch: @escaping (Event) -> Void) {
```

Comment:
I don't think we need `metricsBuilder` passed into init() function?

Reply:
Update function comments to include additional parameters.

Reply:
Yes, I can make metricsBuilder public for testing actually.